AsyncDisplayKit/ASImageNode.mm

```diff
@@ -110,17 +109,11 @@ - (instancetype)initWithViewBlock:(ASDisplayNodeViewBlock)viewBlock didLoadBlock
   return nil;
 }
 
-- (void)dealloc
```

Comment:
@johnepinterest do you remember why this was removed?  I think we'll add it back in.